skimage/typing/_image_types.py

```diff
@@ -0,0 +1,13 @@
+import numpy as np
+from typing_extensions import Annotated
+
+Image3D = Annotated[np.ndarray, 'image_3d',
+                        'a 2D or 3D single or multichannel image']
+Image2D = Annotated[np.ndarray, 'image_2d',
+                        'a 2D single or multichannel image']
+Image = Annotated[np.ndarray, 'image',
+                            'nd-dimensional image']
```

Comment:
```suggestion
                            'an nD image']
```
to mimic the previous descriptions, or 'n-dimensional image'